modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,76 +83,20 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Download, Install, and Configure Istio
 
-You can install Istio 1.6.x in your environment to support Anypoint Service Mesh. However, the Istio configuration for Anypoint Service Mesh differs based on the Istio version that you installed.
+You can install Istio 1.6.x or 1.7.x in your environment to support Anypoint Service Mesh.
 
 === Prerequisites
 
 Before you begin, ensure that you download Istio using the https://istio.io/docs/setup/[Istio Documentation].
 
-=== Install and Configure Istio 1.6.x
+=== Install and Configure Istio
 
-Anypoint Service Mesh supports Istio 1.6.x versions `1.6.0` through `1.6.8`. To install and configure Istio 1.6.x for Anypoint Service Mesh:
+Anypoint Service Mesh supports Istio versions `1.6.0` through `1.7.0`. To install and configure it for Anypoint Service Mesh:
```

Comment:
Can you add something like, ASM has been tried through Istio versions 1.6.0 to 1.6.8 and Istio 1.7.0?

Reply:
done!